action/rlp_tx_test.go

```diff
@@ -344,14 +348,331 @@ func TestRlpDecodeVerify(t *testing.T) {
+			356,
+			"9aa470cbdc3b3fd8f51aae4770d6a58cf4016be18201f0efbae6d83d0b2aa096",
+			"04dc4c548c3a478278a6a09ffa8b5c4b384368e49654b35a6961ee8288fc889cdc39e9f8194e41abdbfac248ef9dc3f37b131a36ee2c052d974c21c1d2cd56730b",
+			"1e14d5373e1af9cc77f0032ad2cd0fba8be5ea2e",
+		},
+		{
+			"candidateUpdate",
+			"f9010609808252089404c22afae6a03438b8fed74cb1cf441168df3f1280b8a4435f9f2200000000000000000000000000000000000000000000000000000000000000600000000000000000000000003041a575c7a70021e3082929798c8c3fdaa9d8240000000000000000000000003041a575c7a70021e3082929798c8c3fdaa9d824000000000000000000000000000000000000000000000000000000000000000474657374000000000000000000000000000000000000000000000000000000008224c6a04df6545da871debef84476198e76167d4dfe6fb83098a5a49dbab734352f20f1a0220686494f8b09751bbeb5c63729e7ecc5c504f952c67e926461db98588854b6",
+			9,
+			21000,
+			"0",
+			"0",
+			"0x04C22AfaE6a03438b8FED74cb1Cf441168DF3F12",
+			_evmNetworkID,
+			iotextypes.Encoding_ETHEREUM_EIP155,
+			164,
+			"7a8d96d35c939bf1634d587b1f471c0f3f96ba750d64d43289c9eef267718ef0",
+			"04dc4c548c3a478278a6a09ffa8b5c4b384368e49654b35a6961ee8288fc889cdc39e9f8194e41abdbfac248ef9dc3f37b131a36ee2c052d974c21c1d2cd56730b",
+			"1e14d5373e1af9cc77f0032ad2cd0fba8be5ea2e",
+		},
+		{
+			"rewardingClaim",
+			"f8c6806482520894a576c141e5659137ddda4223d209d4744b2106be80b8642df163ef0000000000000000000000000000000000000000000000000000000000000065000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000008224c6a03d62943431b8ca0e8ea0a9c79dc8f64df14c965ca5486da124804013778ed566a065956b185116b65cec0ec9bcf9539e924784a4b448190b0aa9d017f1719be921",
+			0,
+			21000,
+			"100",
+			"0",
+			"0xA576C141e5659137ddDa4223d209d4744b2106BE",
+			_evmNetworkID,
+			iotextypes.Encoding_ETHEREUM_EIP155,
+			100,
+			"4d26d0736ebd9e69bd5994f3730b05a2d48c810b3bb54818be65d02004cf4ff4",
+			"04830579b50e01602c2015c24e72fbc48bca1cca1e601b119ca73abe2e0b5bd61fcb7874567e091030d6b644f927445d80e00b3f9ca0c566c21c30615e94c343da",
+			"8d38efe45794d7fceea10b2262c23c12245959db",
+		},
+		{
+			"rewardingDeposit",
+			"f8c6016482520894a576c141e5659137ddda4223d209d4744b2106be80b86427852a6b0000000000000000000000000000000000000000000000000000000000000065000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000008224c6a013b7679dbabcb0f97b93942436f5072cca3c7fe43451a8fedcdf3c84c1344e1da02af4cc67594c0200b59f4e30ba149af15e546acbfc69fa31f14e8788ab063d85",
+			1,
+			21000,
+			"100",
+			"0",
+			"0xA576C141e5659137ddDa4223d209d4744b2106BE",
+			_evmNetworkID,
+			iotextypes.Encoding_ETHEREUM_EIP155,
+			100,
+			"842fea9a292c0bc7a1e05a14a8255ed8dc945fdae7c01a6b70f5213ebe35583b",
+			"04830579b50e01602c2015c24e72fbc48bca1cca1e601b119ca73abe2e0b5bd61fcb7874567e091030d6b644f927445d80e00b3f9ca0c566c21c30615e94c343da",
+			"8d38efe45794d7fceea10b2262c23c12245959db",
+		},
+		{
```

Comment:
what error if integrated with `TestRlpDecodeVerify``

Reply:
see `func TestBackwardComp(t *testing.T)` below, it verifies error will occur running the new tx with current code

Reply:
ok, could testdata be shared in `Var` ?

Reply:
updated in latest commit